### ISSUE TITLE
Zpool cannot create pool / mix multipath with vdev_id.conf defined disks (not a bug)

### ISSUE THREAD

**Issue by ltaulell** [zfsonlinux/zfs#5008]
Not a bug, Maybe FAQ/Documentation lack.

TL;DR: Do NOT mix multipath and vdev_id.conf configured disks.

I have ran into a mix of several already reported bugs, which were all closed (and patched/corrected) on a brand new server (2016/07, Debian Jessie, zfs 0.6.5.7-8-jessie).

We install our debian fileservers with a bunch of packages, regardless of usage. This include multipath and debian-zfs.

On this particular server (Dell R730xd), there is only one standard SAS card (H330, in JBOD mode), with one backplane and 24+2 disks. No multipath needs. So I configured it only with vdev_id.conf file [r730datacs-vdev_id.conf.txt](https://github.com/zfsonlinux/zfs/files/430357/r730datacs-vdev_id.conf.txt).

I ran into, what seems to be #2582, #2493, and #3708 bugs (at least): I was unable to create any zpool, unless:

 * purge all multipath packages,
 * manually add a gpt label on each futur-zfs disks,
 * reboot (reread partitions tables was not enough, /dev/mapper need to be empty, apparently).

The thing that lead me to a multipath "problem" was lsblk, showing "TYPE: mpath" instead of "TYPE: disk" one some disks (not all), although I was not using multipath at all (no config).

Now, it's running fine. I've been able to create some pools, even use them :o)

Hope its helps.

**Comment by kpande** [comment]
please use the mailing list for discussion in the future, since this isnt a bug.
